lib/api.js

```diff
@@ -63,14 +65,17 @@ function _prepareOptionsFor(method, options) {
     ].join(''),
   };
   if (options['omiseVersion']) {
-    headers['Omise-Version'] = options['omiseVersion'];
+    headers['omiseVersion'] = options['omiseVersion'];
```

Comment:
`Omise-Interface` read version header in a `Omise-Version` form. It need to be either `Omise-Version` or `omise-version`. `omiseVersion` won't be readed.

Reply:
Changed back to Omise-Version